platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -407,23 +409,34 @@ private MigrationReport doDecrypt(
   private void restart(MigrationReport report) {
     try {
       if (!restartServiceWrapperIfControlled()) {
-        LOGGER.debug("asking karaf to restart");
-        System.setProperty("karaf.restart.jvm", "true");
-        system.reboot();
+        // create the restart.jvm file such that we would rely on the ddf script to restart
```

Comment:
Same comments here as the ones provided in the `ApplicationServiceBean` class above.

Reply:
same formatter issue ... I hate it.